backend/de.metas.business/src/main/java/de/metas/invoice/interceptor/C_Invoice.java

```diff
@@ -177,17 +177,20 @@ public void removeMaterialLinesNotCorrespondingToPriceList(final I_C_Invoice inv
 	@CalloutMethod(columnNames = I_C_Invoice.COLUMNNAME_C_BPartner_ID)
 	public void setPaymentRule(final I_C_Invoice invoice)
 	{
-		final I_C_BPartner bpartner = bpartnerDAO.getById(invoice.getC_BPartner_ID());
-		final PaymentRule paymentRule;
-		if (bpartner != null && bpartner.getPaymentRule() != null)
+		if (!InterfaceWrapperHelper.isCopying(invoice) && !InterfaceWrapperHelper.isNew(invoice))
```

Comment:
please add javadoc to explain why this method only acts if (!InterfaceWrapperHelper.isCopying(invoice) && !InterfaceWrapperHelper.isNew(invoice))`

❗ btw, what about the case that a new invoice without an order is created in the webui?